.github/workflows/linkcheck.yml

```
@@ -21,11 +40,7 @@ jobs:
         uses: lycheeverse/lychee-action@v1.10.0
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
-          args: > 
-          --scheme https --scheme http --verbose --no-cache
-          --user-agent "Mozilla/5.0 (Windows NT 10.0; Win64; x64)"
-          --max-concurrency 5 --retry-count 3 --retry-delay 2
-          ${{ steps.changed-files.outputs.all_changed_files }}
+          args: --base content --scheme https --scheme http --verbose --no-cache --user-agent "Mozilla/5.0 (Windows NT 10.0; Win64; x64)" --max-concurrency 5 --retry-count 3 --retry-delay 2 ${{ steps.changed-files.outputs.all_changed_files }}
```

```suggestion
          args: --accept 200,429 --base content --scheme https --scheme http --verbose --cache --user-agent "Mozilla/5.0 (Windows NT 10.0; Win64; x64)" --max-concurrency 5 --retry-count 1 --retry-delay 2 ${{ steps.changed-files.outputs.all_changed_files }} --github-token "${{ secrets.GITHUB_TOKEN }}"
```

This is to reduce this problem: https://lychee.cli.rs/troubleshooting/rate-limits/

---

OK I tried to address this one, your feedback above, and also to add the things needed to use `--cache`, in the most recent commit. I was hoping that setting the cron to a specific time I could watch the Actions tab in the repo and find out if it worked. But maybe the PR has to be committed for that to work. I modeled the change on https://github.com/lycheeverse/lychee-action, which passes the args using a different syntax than we were using previously.